TASK-4/PersonsRepository.cs

```diff
@@ -0,0 +1,38 @@
+using Bogus;
+
+namespace Persons;
+
+public class PersonsRepository
+{
+    private readonly string[] companyNames = { "Z", "Apple", "Yandex", "Google" };
+    private readonly string[] desiredPositions = { "Dynamic Identity Engineer",
+      "Dynamic Intranet Facilitator","Global Creative Engineer", "Internal Markets Agent", "International Accounts Representative" };
+
+    public IEnumerable<Candidate> GetCandidates(List<Candidate> listOfObjects)
+    {
+        var candidateGenerator = new Faker<Candidate>()
+            .RuleFor(c => c.Id, Guid.NewGuid())
+            .RuleFor(c => c.Name, (f, c) => f.Name.FirstName())
+            .RuleFor(c => c.Surname, (f, c) => f.Name.LastName())
+            .RuleFor(c => c.DesiredPosition, f => f.PickRandom(desiredPositions))
+            .RuleFor(c => c.PositionDescription, (f, c) => f.Name.JobDescriptor())
+            .RuleFor(c => c.DesiredSalary, f => f.Random.Double(500, 10000));
```

Comment:
This values should be declared in the top of the class, it is hardcode when you use them more than 1 time

Reply:
Ok, moved to the top.